README.md

```diff
@@ -693,9 +693,13 @@ Other options:
  - `toplevel` — mangle names declared in the toplevel scope (disabled by
   default).
 
-  - `eval` — mangle names visible in scopes where eval or with are used
+   - `eval` — mangle names visible in scopes where eval or with are used
   (disabled by default).
 
+ - `keep_fnames` -- default `false`.  Pass `true` to not mangle
```

Comment:
`eval` is still incorrectly indented. I appreciate this problem existed prior to your pull request. 
It can be seen with the View button above: https://github.com/homuler/UglifyJS2/blob/30f718fe0d01b10a92e2f43ccf6b68b19aa55186/README.md 
Please line up the dash on the keep_fnames line to match previous lines. That may fix it.